ui/utils/lists.ts

```diff
@@ -16,4 +16,8 @@ export const intersperseWith = <T, K>(
   return result
 }
 
+export const excludeFalsyValues = <T>(items: (T | boolean)[]): T[] => {
+  return items.filter((item): item is Exclude<T | boolean, boolean> => !!item)
+}
```

Comment:
Sweet! Just a small nit, we can remove the `boolean` union from the parameter as it is implied that T can be a `boolean`, we should also fix the return type here.

```suggestion
export const excludeFalsyValues = <T>(items: T[]): Exclude<T, boolean>[] => {
  return items.filter((item): item is Exclude<T, boolean> => !!item)
}
```

Reply:
It is no longer used.